src/test/java/org/jenkinsci/plugins/github/config/GitHubServerConfigTest.java

```diff
@@ -74,7 +75,7 @@ public void shouldGuessNameIfNotProvided() throws Exception {
         GitHubServerConfig input = new GitHubServerConfig("");
         input.setApiUrl(CUSTOM_GH_SERVER);
         assertThat(input.getName(), is(nullValue()));
-        assertThat(input.getDisplayName(), is("some (http://some.com)"));
```

Comment:
what's the reason for this change?

Reply:
I can't tell as it's working as well. @Vlatombe any reason behind this change?

Reply:
Looks like the test is failing with `mvn test -Dtest=GitHubServerConfigTest -Djenkins.version=2.332`. Looking up why but it looks related to guava upgrade.

Reply:
There is a swallowed error when running against jenkins 2.332

```
java.lang.NoSuchFieldError: EXACT
	at com.google.common.net.InternetDomainName.findSuffixOfType(InternetDomainName.java:171)
	at com.google.common.net.InternetDomainName.<init>(InternetDomainName.java:152)
	at com.google.common.net.InternetDomainName.from(InternetDomainName.java:210)
	at jenkins.scm.api.SCMName.resolveInternetDomainName(SCMName.java:143)
	at jenkins.scm.api.SCMName.innerFromUrl(SCMName.java:113)
	at jenkins.scm.api.SCMName.fromUrl(SCMName.java:67)
	at org.jenkinsci.plugins.github.config.GitHubServerConfigTest.given__url_with(GitHubServerConfigTest.java:83)
[...]
```

This is caused by the `wiremock:standalone` dependency which is vendoring `com.google.thirdparty.publicsuffix.PublicSuffixPatterns`

Reply:
The proper fix would be to remove the wiremock `standalone` classifier once the Jenkins baseline is raised to 2.332 or later. Until then, this test change seems to be the only way to run tests with both baselines (for PCT runs purpose against recent Jenkins baselines)

Reply:
You should always use the wiremock standalone due to the mass of library conflicts- however the version used is very old and may not be correctly packaged.

should use 
```
<dependency>
  <groupId>com.github.tomakehurst</groupId>
  <artifactId>wiremock-jre8-standalone</artifactId>
  <version>xxx</version>
</dependency>
```

Reply:
I upgraded wiremock, looks good.